src/main.rs

```diff
@@ -90,13 +91,40 @@ fn retrieve_doc_comment(allow_single_line: bool, meta: &Meta) -> Option<String>
 
 /// Transforms an AST node into a `DocItem` if it has a leading
 /// documentation comment.
-fn retrieve_doc_item(node: &ASTNode) -> Option<DocItem> {
+fn retrieve_doc_item<'a>(arena: &Arena<'a>, node: &ASTNode) -> Option<DocItem> {
```

Comment:
According to clippy these lifetimes can be elided.

```suggestion
fn retrieve_doc_item(arena: &Arena, node: &ASTNode) -> Option<DocItem> {
```

Reply:
Thanks, updated!